src/reducers/search.js

```diff
@@ -38,6 +38,7 @@ export const initialState = {
     displayName: 'Top inspiration',
     id: 'marketing:homepage.dk.spies'
   },
+  scrollPage: 6,
```

Comment:
30 items loaded initially and 5 per page so when the user scrolls to the bottom of the first page the page is not reloaded with 5 items.